src/main/java/io/deepstream/Actions.java

```diff
@@ -141,7 +141,11 @@
     /**
      * Used to query for clients
      */
-    QUERY( "Q" );
+    QUERY( "Q" ),
+    /**
+     * Used when requiring write acknowledgements when setting records
+     */
+    WRITE_SUCCESS( "WS" );
```

Comment:
write acknowledgement

action names need to be constant in both server and client ( code practice )